electron/legendary/games.ts

```diff
@@ -403,16 +403,29 @@ class LegendaryGame extends Game {
       )
     }
 
-    const res = await runLegendaryCommand(commandParts, {
+    let res = await runLegendaryCommand(commandParts, {
       logFile: logPath,
       onOutput
     })
 
-    if (res.error) {
-      logError(
-        ['Failed to install', `${this.appName}:`, res.error],
-        LogPrefix.Legendary
+    // try to run the install again with higher memory limit
+    if (res.error === 'MemoryError:') {
+      res = await runLegendaryCommand(
+        [...commandParts, '--max-shared-memory', '5000'],
```

Comment:
I'd say it would be better to obtain the shared memory requirement from the legendary output, even 5000MiB might not be enough for some games

Reply:
I think will work for 99% of the games. I mean, I never had any issue with this even before Heroic, when using legendary only. So its an edge case deal with the string output is too boring.

Reply:
Fair enough
It would be interesting to know why Legendary doesn't do that by default, if it has no negative effects